CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -696,6 +732,94 @@ private void GenerateLogarithmicArithmeticOperators()
 " );
         }
 
+        /// <summary>
+        ///     Generates operators that express relations between quantities as applied by <see cref="QuantityRelationsParser" />.
+        /// </summary>
+        private void GenerateRelationalOperators()
+        {
+            if (!_quantity.Relations.Any()) return;
+
+            Writer.WL($@"
+        #region Relational Operators
+");
+
+            foreach (QuantityRelation relation in _quantity.Relations)
+            {
+                if (relation.Operator == "inverse")
+                {
+                    Writer.WL($@"
+        /// <summary>Calculates the inverse of this quantity.</summary>
+        /// <returns>The corresponding inverse quantity, <see cref=""{relation.RightQuantity.Name}""/>.</returns>
+        public {relation.RightQuantity.Name} Inverse()
+        {{
+            return {relation.LeftUnit.PluralName} == 0.0 ? {relation.RightQuantity.Name}.Zero : {relation.RightQuantity.Name}.From{relation.RightUnit.PluralName}(1 / {relation.LeftUnit.PluralName});
+        }}
+");
+                }
+                else
+                {
+                    var leftParameter = relation.LeftQuantity.Name.ToCamelCase();
+                    var leftConversionProperty = relation.LeftUnit.PluralName;
+                    var rightParameter = relation.RightQuantity.Name.ToCamelCase();
+                    var rightConversionProperty = relation.RightUnit.PluralName;
+
+                    if (relation.LeftQuantity.Name == "TimeSpan")
```

Comment:
Maybe reuse some constants for all these repeating strings

Reply:
I tried to tidy up a bit, but code generation is always a bit messy and throwing around consts called `_double` and `_value` didn't make things better 😅